.editorconfig

```diff
@@ -0,0 +1,15 @@
+# editorconfig for VS code
+
+root = true
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{c,h,cc}]
```

Comment:
Please see if you can make this list more comprehensive by reviewing _all_ filename extensions used by Squid sources. The rules below apply to most Squid sources with Makefile and cf.data.pre being notable exceptions (that I recall).